src/fal/cli.py

```diff
@@ -50,7 +50,15 @@ def run(run, project_dir, profiles_dir, keyword, all, debug):
 
         project = parse_project(real_project_dir, real_profiles_dir, keyword)
         models = project.get_filtered_models(all)
-        ordered_scripts = ScriptGraph(models, keyword, project_dir).sort()
+
+        models_arr = map(
+            lambda model: f"{model.name}: {', '.join(model.meta['fal']['scripts'])}",
+            models
+        )
+        models_str = '\n'.join(models_arr)
+        print_timestamped_line(f"Starting FAL run for following models (model_name: scripts): \n{models_str}")
```

Comment:
can you move this its own function and outside of this file, we keep adding more logic here and trying to keep cli.py interface to the cli